packages/extension-vscode/src/server.ts

```diff
@@ -28,6 +56,12 @@ connection.onDidChangeWatchedFiles(async () => {
     }));
 });
 
+connection.onCodeAction(quickFixActionProvider.provideCodeActions.bind(quickFixActionProvider));
+
+connection.onExecuteCommand(() => {
+    return;
```

Comment:
Is this left-over code or does having this registration cause some sort of side-effect?